vendor/github.com/minio/minio-go/post-policy.go

```diff
@@ -167,6 +167,28 @@ func (p *PostPolicy) SetSuccessStatusAction(status string) error {
 	return nil
 }
 
+// SetUserMetadata - Set user metadata as a key/value couple.
+// Can be retrieved through a HEAD request or an event.
+func (p *PostPolicy) SetUserMetadata(key string, value string) error {
+	if strings.TrimSpace(key) == "" || key == "" {
```

Comment:
`strings.TrimSpace(key)` for `key=""` returns `""`. 
So, the 2nd condition is covered in the 1st part. 
Then we can simplify the if condition as:
 ```
if strings.TrimSpace(key) == "" {
```